net/crowdsec/files/crowdsec.initd

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh /etc/rc.common
+# (C) 2021 Gerald Kerma
+
+START=99
+USE_PROCD=1
+NAME=crowdsec
+PROG=/usr/bin/crowdsec
+CONFIG=/etc/crowdsec/config.yaml
+RUNCONFDIR=/srv/crowdsec/data
+
+service_triggers() {
+        procd_add_reload_trigger crowdsec
+}
+
+boot()
+{
+	rc_procd start_service
+}
+
+init_config() {
+	config_load crowdsec
+	config_get data_dir crowdsec data_dir "${RUNCONFDIR}"
+	config_get db_path crowdsec db_path "${RUNCONFDIR}/crowdsec.db"
+	
+        sed -i "s,^\(\s*data_dir\s*:\s*\).*\$,\1$data_dir," $CONFIG
+        sed -i "s,^\(\s*db_path\s*:\s*\).*\$,\1$db_path," $CONFIG
```

Comment:
Is there a reason why these lines are indented differently?

Reply:
fixed
thanks